runtime/compiler/x/i386/codegen/IA32JNILinkage.cpp

```diff
@@ -491,7 +313,7 @@ TR::Register *TR::IA32JNILinkage::buildJNIDispatch(TR::Node *callNode)
       );
 
    if (isJNIGCPoint)
-      instr->setNeedsGCMap((argSize<<14) | getProperties().getPreservedRegisterMapForGC());
+      instr->setNeedsGCMap((argSize<<14) | 0xFF00FFFF);
```

Comment:
could you explain why is this change needed here? What's the meaning of the MASK? Is it possible to give it a meaningful name to indicate its purpose?